.github/workflows/integrationTest.yml

```diff
@@ -158,6 +160,8 @@ jobs:
       contents: read
     steps:
       - uses: actions/checkout@v2
+        with:
+          repository: aws/amazon-cloudwatch-agent-test
```

Comment:
Would it be better to use `path` option which would reduce less changes and raise more awareness when using the integration test repository?

Reply:
What is that good for?

Reply:
Its create a [relative path for the checkout repository](https://github.com/actions/checkout#usage). For example, by using `path:integration`, you don't need to delete all the integration path and it differentiate between checkout the main repo and the integration test repo

Reply:
integration directory is already deleted in the test package

Reply:
Is there any value in re-making changes to revert them all both in here and in test package?

Reply:
Not much besides raise awareness but should not block the PR from getting merged. So I'm good with either options, only want to provide options.